Content.Shared/GameObjects/Components/Observer/GhostWarpToNameRequestMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.Components.Observer
+{
+    [Serializable, NetSerializable]
+    public class GhostWarpToNameRequestMessage : ComponentMessage
+    {
+        public GhostWarpToNameRequestMessage(string warpTarget = default)
+        {
+            WarpName = warpTarget;
+            Directed = true;
+        }
+
+        public string WarpName { get; }
```

Comment:
I feel like it's more clean to put this above the constructor.

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3310/commits/712431c27a251123feeed82cf066580a24dce43e